CONTRIBUTING.md

```diff
@@ -155,9 +181,12 @@ structure, mechanics and philosophy.
 
 You can run tests directly from VSCode:
 
-1. Select `View > Debug`, or select the Debug pane from the sidebar.
-2. From the dropdown at the top of the Debug pane, select the `Extension Tests` configuration.
-3. Press `F5` to run tests with the debugger attached.
+1. Go to the `Run & Debug` menu.
+2. Select the correct config to run the correct test
+    - (Red Arrow) Regular unit tests in `src/test/`
+    - (Blue Arrow) Integration and E2E tests in `src/testInteg/` and `src/testE2E/` respectively.
+3. Click the green arrow to run  
+   <img src="./docs/images/runTests.png" alt="Developer Mode Enabled" width="512"/>
```

Comment:
This image will get out of date quickly. Let's instead make the test names more self-explanatory (without being verbose), and try to eliminate some of the less valuable ones. For example we could rename `Extension Tests` to `Test (src/test/)`, `Integration Tests` to `Test Integ (/src/testInteg/)`, etc.